app/src/main/java/fr/free/nrw/commons/auth/LoginActivity.java

```diff
@@ -97,6 +98,18 @@ public void onCreate(Bundle savedInstanceState) {
         binding = ActivityLoginBinding.inflate(getLayoutInflater());
         setContentView(binding.getRoot());
 
+
+        //Get Extra message string from the intent and show Toast
+        String message = getIntent().getStringExtra("loginMessage");
+        String username = getIntent().getStringExtra("username");
+        if(message!=null) {
```

Comment:
```suggestion
        if (message != null) {
```

Throughout this PR, please can you follow the Google style guide: https://github.com/commons-app/commons-app-documentation/blob/master/android/Code-style.md